wechatpay/src/main/java/com/adyen/checkout/wechatpay/WeChatPayActionComponent.java

```diff
@@ -70,27 +67,15 @@ public WeChatPayActionComponent(@NonNull Application application, @NonNull WeCha
      *
      * @param intent The intent result from WeChatPay SDK.
      */
-    public void handleResultIntent(@Nullable Intent intent) {
```

Comment:
Don't forget to add this to the release notes since it's a breaking change for merchants using the Component only.

Reply:
Done 👍  Pushed 3 new commits